scene/3d/mesh_instance_3d.cpp

```diff
@@ -383,13 +383,13 @@ void MeshInstance3D::_mesh_changed() {
 	update_gizmos();
 }
 
-void MeshInstance3D::create_debug_tangents() {
+MeshInstance3D *MeshInstance3D::create_debug_tangents_node() {
```

Comment:
I would suggest making create_debug_tangents return the mesh instance it creates and adds as a child, but I think that would require me to open a GIP